index.js

```diff
@@ -1,15 +1,27 @@
 'use strict'
 
+const cp = require('child_process')
 const path = require('path')
 const stream = require('stream')
 const browserify = require('browserify')
 const extend = require('extend-shallow')
 
 const Readable = stream.Readable
 
+const browserifyPath = path.resolve(__dirname, 'lib', 'browserify.js')
+
 exports.name = 'browserify'
 exports.outputFormat = 'js'
 
+exports.renderFile = function (filename, options, locals) {
+  // Spawn a browserify process synchronously
+  const output = cp.execSync(
+    `node ${browserifyPath} file ${stringifyForCli(filename)} ${stringifyForCli(options)} ${stringifyForCli(locals)}`
```

Comment:
Great solution! Does `execFileSync` process directly with node? Do we need the `node` prefix? I suppose if the platform doesn't register it globally, it may break. Is there a way to discover the node executable?

Reply:
There is a way to convert it to the bin file (by adding `#!/usr/bin/env node` at the start) and run it as an executable. I think this will do the job. I will change the file if you're good with this :)

Reply:
That'd be cool, I guess the file itself would have to be executable permissions though, right? Using `node` is fine for now.

Anything missing before this is merged?!?!?!

Reply:
I guess it would, you're right. Anyway, I think `node` would be just fine. Nope, nothing missing :)